### PR TITLE
Added unit test, travis file

### PR DESCRIPTION
**Pull request by tleeuwenburg**
I should have split this up into multiple commits, but..

-- Added 'travis.yml' which can be used to integrate travis (done for my fork)
-- Removed failing test file completely as you suggested
-- Added a single unit test covering the softmax function (to show the approach)

### CODE REVIEW

**Comment by tleeuwenburg** [comment]
Note -- this failed somehow after I merged in your master branch -- hold off on this for now until I've fixed it

**Comment by tleeuwenburg** [comment]
Okay, it works again now, plus I added a test for the linear activation function

**Comment by phreeza** [comment]
This looks great. I am motivated to help working on comprehensive unit tests, maybe we should set up a trello for that?

**Comment by fchollet** [comment]
Alright, let's merge this.

> I am motivated to help working on comprehensive unit tests

Awesome!

> maybe we should set up a trello for that?

Wouldn't Github issues suffice? I find it easier to have everything in one place.

**Comment by tleeuwenburg** [comment]
I think the main thing is to work on separate files so as not to step on
eachother's toes. I don't think it's critical to work using Trello.

Writing tests can sometimes benefit a lot from a bit of refactoring. Do we
want a two-step process that includes a code review? I'd be happy to give
you commit access to my fork, and offer my services as a review/refactorer
:)

Otherwise, we can just go at it on trunk and try to work in separate areas.

It might be nice working together though -- more motivating :)

On 27 June 2015 at 11:13, François Chollet <notifications@github.com> wrote:

> Alright, let's merge this.
>
> I am motivated to help working on comprehensive unit tests
>
> Awesome!
>
> maybe we should set up a trello for that?
>
> Wouldn't Github issues suffice? I find it easier to have everything in one
> place.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fchollet/keras/pull/280#issuecomment-115931156>.
>



-- 
--------------------------------------------------
Tennessee Leeuwenburg
http://myownhat.blogspot.com/
"Don't believe everything you think"
